shoplist/src/main/java/com/netguru/codereview/network/model/ShopListResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListResponse(
+    val list_id: String,
```

Comment:
As we are following the "camelCase" naming convention, please use SerializedName to parse the response if needed. or just rename it to listId instead
```suggestion
@SerialName("list_id")
    val listId: String,
```